File: TransCoder_tokenized_test_set_functions/python/NEWMAN_CONWAY_SEQUENCE_1.py
def sequence ( n ) :
    f = array . array ( 'i', [ 0, 1, 1 ] )
    for i in range ( 3, n + 1 ) :
        r = f [ f [ i - 1 ] ] + f [ i - f [ i - 1 ] ]
        f . append ( r );
        
    return r

File: TransCoder_tokenized_test_set_functions/cpp/NEWMAN_CONWAY_SEQUENCE_1.cpp
int sequence ( int n ) {
  int f [ n + 1 ];
  int i;
  f [ 0 ] = 0;
  f [ 1 ] = 1;
  f [ 2 ] = 1;
  for ( i = 3;
  i <= n;
  i ++ ) f [ i ] = f [ f [ i - 1 ] ] + f [ i - f [ i - 1 ] ];
  return f [ n ];
}